src/dlgSourceEditorArea.cpp

```diff
@@ -20,14 +20,55 @@
 
 
 #include "dlgSourceEditorArea.h"
+#include "edbee/edbee.h"
+#include "edbee/texteditorwidget.h"
+#include "edbee/models/textdocument.h"
+#include "edbee/views/textrenderer.h"
+#include "edbee/views/texttheme.h"
+#include "edbee/models/textgrammar.h"
+#include "edbee/models/texteditorconfig.h"
 
-#include "THighlighter.h"
+#include <QDir>
+
+
+//#include "THighlighter.h"
 
 
 dlgSourceEditorArea::dlgSourceEditorArea(QWidget* pF) : QWidget(pF)
 {
     // init generated dialog
     setupUi(this);
-    highlighter = new THighlighter(editor->document());
-    editor->setTabStopWidth(25);
+
+    //highlighter = new THighlighter(editor->document());
+    //editor->setTabStopWidth(25);
+
+    // Setting up the edbee widget
+
+    edbee::Edbee* ei = edbee::Edbee::instance();
+
+    ei->autoInit();
+    ei->autoShutDownOnAppExit();
+
+    ei->setGrammarPath(  "G:\\Temp\\edbee\\syntaxfiles\\" );
+    ei->setThemePath( "G:\\Temp\\edbee\\themes\\" );
+
+    edbee::TextGrammarManager* grammarManager = ei->grammarManager();
+    edbee::TextGrammar* grammar = grammarManager->readGrammarFile( QDir::homePath() + "/.config/mudlet/edbee/Lua.tmLanguage" );
+    edbee::TextThemeManager* themeManager = ei->themeManager();
+
+    edbee::TextTheme* theme = themeManager->readThemeFile( QDir::homePath() + "/.config/mudlet/edbee/Fluidvisionlet.tmTheme" );
+    edbeeEditorWidget->config()->setThemeName("Fluidvisionlet");
+
+    edbeeEditorWidget->config()->setShowCaretOffset( true);
+    edbeeEditorWidget->config()->setCaretWidth( 1);
+    //edbeeEditorWidget->config()->setExtraLineSpacing(40);
+    edbeeEditorWidget->config()->setSmartTab( true);
+    edbeeEditorWidget->config()->setCaretBlinkRate( 200);
+
+    edbeeEditorWidget->textDocument()->setLanguageGrammar( grammar );
+    edbeeEditorWidget->config()->setIndentSize( 2);
+    edbeeEditorWidget->config()->setFont(QFont( "Courier New", 28));
```

Comment:
The font is already following the configured font - the setting to Courier New is just a fallback.